cypress/features/regression/designSystem/toast.feature

```diff
@@ -0,0 +1,71 @@
+Feature: Toast Design System component
+  I want to test Design System Toast component properties
+
+  Background: Open Design System Toast component default page
+    Given I open Design Systems default_story "Toast" component docs page
+
+  @positive
+  Scenario: Verify the click action in Actions Tab
+    Given I open basic Test "Toast" component page
+    When clear all actions in Actions Tab
+      And I click closeIcon
+    Then click action was called in Actions Tab
+
+  @positive
+  Scenario Outline: Change Toast children to <children>
```

Comment:
move this TC to `\cypress\features\regression\test\toast.feature` becouse you use knob